common/reactive/src/main/java/io/helidon/common/reactive/Awaitable.java

```diff
@@ -64,6 +64,7 @@ default T await() {
      * @throws java.util.concurrent.CompletionException   if this future completed exceptionally,
      *                                                    was interrupted while waiting or the wait timed out
      */
+    @Deprecated
```

Comment:
Whenever `@Deprecated` is used, the following rules must be followed:
- add `since` to clearly mark when we added this
- if the method is expected to be removed, as `forRemoval = true`
- add `@deprecated` javadoc tag with information what to do

So for this example:

```java
 * @deprecated use {@link #await(Duration)} instead
 */
 @Deprecated(since = "4.0.0")
```